base/io.jl

```diff
@@ -448,7 +456,29 @@ function readuntil(s::IO, t::AbstractString)
 end
 
 readline() = readline(STDIN)
-readline(s::IO) = readuntil(s, '\n')
+
+function readline(s::IO, chomp::Bool=false)
```

Comment:
It's still needed. C version reads from IOStreams, but not for instance IOBuffers.